models/pancreas_ct_dints_segmentation/configs/metadata.json

```diff
@@ -31,12 +32,15 @@
     "monai_version": "1.4.0",
```

Comment:
Do we need update the monai version when monai tagged 1.5?

Reply:
HI @KumoLiu , it depends on the users, there is no hard requirements. However, if a bundle is too old (for example, uses a torch version that is not supported by our CI's python version), we may update it.